src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -1474,15 +1475,17 @@ private void CreateOperationMethods(OpenApiUrlTreeNode currentNode, NetHttpMetho
             {
                 (_, true) => [],
                 (null, _) => operation.Responses!
-                                .WhereValidForMediaTypeSelection()
```

Comment:
[WhereValidForMediaTypeSelection](https://github.com/microsoft/kiota/blob/b23e416d629bac68ecf21076694eee085591745a/src/Kiota.Builder/Extensions/OpenApiResponsesExtensions.cs#L11-L15) extension method is no longer used in the code except for test code. Consider removing this code if there isn't a plan to use it.

https://github.com/microsoft/kiota/blob/main/tests/Kiota.Builder.Tests/Extensions/OpenApiResponsesExtensionsTests.cs